### DISCUSSION TITLE
قوانین مشارکت در محتوا

### DISCUSSION THREAD

**Discussion by Hameds** [softwaretalks/newsletter#107] [General]
<div dir='rtl'>

این نسخه اولیه پیشنهادی من برای قوانین مشارکت در محتواست.

- تا حد امکان عنوان و توضیحات محتوای پیشنهادی که در مخزن در قالب `issue` اضافه می‌شود، کوتاه و به زبان فارسی باشند. البته خود محتوا می‌تواند به فارسی یا انگلیسی باشد.
- لینک محتوای پیشنهادی نباید از سرویس‌های کوتاه‌کننده‌های لینک باشد مگر کوتاه کننده لینک مربوط به سرویس‌دهنده همان محتوا
- لینک محتوای پیشنهادی می‌بایست از خارج از ایران هم در دسترس باشد، محتوایی که فقط با IP ایران در دسترس است مورد پذیرش نیست
- در صورتی که سایت محتوای پیشنهادی در حال فروش مستقیم محصول یا خدمات است، محتوا باید خارج از بخش تبلیغاتی و فروش سایت باشد
- محتوای بازاریابی محصولات نرم‌افزاری مورد تایید برای ارسال در خبرنامه نیست
- از صفحه هر شخص (مثال ویرگول یا یوتوب یا وبلاگ شخصی و امثالهم) در هر شماره فقط یک محتوا برای ارسال در خبرنامه تایید می‌شود اما محدودیتی در ارسال لینک‌های متعدد نیست
- لینک یا محتوای تکراری در خبرنامه‌های پیشین مجدداً ارسال نمی‌شود
-  چنانچه محتوای پیشنهادی مربوط به مسائل روابط عمومی یک شرکت است، با صلاحدید مدیران و تنها از وب سایت رسمی آن شرکت امکان پیشنهاد محتوا وجود دارد و نظرات کارمندان در صفحات شخصی معیار پذیرش محتوای مرتبط با روابط عمومی شرکت‌ها نیست
- بدیهی است مسئولیت صحت اطلاعات در هر محتوای پیشنهادی به عهده نویسنده محتوا می‌باشد.
- هر کاربری می‌تواند محتواهای ایجاد شده توسط خودش یا دیگران را که شرایط بالا را داشته باشند به تعداد نامحدود در این مخزن معرفی کند  

</div>

**Comment by ohmydevops** [comment]
<div dir='rtl'>
سلام. 
این بند که گفته شده:
</div>

> چنانچه محتوای پیشنهادی مربوط به مسائل روابط عمومی یک شرکت است، با صلاحدید مدیران و تنها از وب سایت رسمی آن شرکت امکان پیشنهاد محتوا وجود دارد و نظرات کارمندان در صفحات شخصی معیار پذیرش محتوای مرتبط با روابط عمومی شرکت‌ها نیست

<div dir='rtl'>
بیشتر توضیحش میدین؟
@Hameds 
</div>

**Reply by Hameds** [reply]
هر شرکت از تعدادی کارمند تشکیل شده، اظهار نظر رسمی درباره موارد یک شرکت به عهده روابط عمومی و از وب سایت رسمی آن شرکت است. فرض کنید خبری درباره نشت اطلاعات یک شرکت وجود داره، اگر توضیحی هست باید از وب سایت رسمی آن شرکت باشه نه توضیحات کارمندان شرکت در شبکه‌های اجتماعی یا وبلاگ‌ها/وب‌سایت‌های شخصی. اگر ادبیاتش نیاز به اصلاح داره بفرمایید

**Reply by ohmydevops** [reply]
الان کامل برام شفاف شد. بسیار نکته مهم و خوبیه.

**Comment by ohmydevops** [comment]
<div dir='rtl'>
درباره مسائل مذهبی/سیاسی که ممکنه بعضی محتواها داشته باشند، باید یک رویکرد مشخص داشته باشیم.
</div>

**Reply by Hameds** [reply]
مثال می‌زنید؟ مثلاً محتوای توسعه نرم‌افزار چطور می‌تونه به مسائل مذهبی/سیاسی مرتبط باشه؟

**Reply by ohmydevops** [reply]
راستش مثال خاصی الان به ذهنم نمیرسه ولی اگر زمانی محتوایی در راستای ترویج مباحث این مدلی ارسال بشه باید رویکردمون مشخص باشه. ولی خب احتمالش نزدیک به صفره :smile:

**Comment by ohmydevops** [comment]
<div dir='rtl'>

یک مساله دیگه کاربرانی هستند که محتواهایی از جنس How-to به اشتراک میزارن.
از دید من How-to ها دو دسته‌اند:

- دسته بدیهیات هر حوزه
- دسته چیزهای جالب و باحال و آموزنده

مثال میزنم برای هر دو مورد:

- نصب داکر: از دید من نصب داکر در شاخه بدیهیات حوزه خودش قرار میگیره. اگر کاربری همچین مطلبی (فارسی یا انگلیسی) به اشتراک بگذاره چه رویکردی خواهیم داشت؟
- راه‌حلی برای ساده کردن شبکه داکر برای ایزوله شدن یک میلیون کانتینر: البته این تیتر رو همین‌طوری نوشتم ولی خب مثلا همچین How-to ای حاصل تجربه یک فرد متخصصه که اگرچه ممکنه چیزی نباشه که جزو روتین‌ها محسوب بشه ولی چیزیه که میشه با خوندنش نحوه تفکر و حل مساله‌های مشابه و کوچکتر رو هم یاد گرفت. رویکرد ما در قبال این مباحث چه خواهد بود؟ 

</div>

**Reply by ohmydevops** [reply]
نظر شخصی من اینه که دسته‌اول یعنی بدیهیات باید حذف بشن از خبرنامه تا کیفیت خبرنامه برای مشترکینش بالا بمونه.

**Comment by ghost** [comment]
به نظرم برای حفظ یکدستی و خوانایی بیشتر خبرنامه، این قانون هم اضافه کنید که محتوا به زبان **فارسی معیار** نوشته بشه. ترکیب فارسی معیار و محاوره‌ای خوب نیست و رسم‌الخط محاوره‌ای به خاطر تنوع زیاد شیوه‌های نگارشی، مناسب نیست.

حتی با فارسی معیار هم سبک نگارش هر فرد متفاوته؛ بنابراین ایده خوبیه قبل از انتشار خبرنامه، محتواش ویراست بشه. اگر خبرنامه ماهیانه بود خودم می‌تونستم این مسئولیت رو بر عهده بگیرم.

**Reply by ohmydevops** [reply]
<div dir='rtl'>

اول از همه ازتون بابت اصلاحیات خبرنامه تشکر میکنم. 
موضوعی که میگید رو کاملا قبول دارم که اگر رعایت بشه یکپارچگی و خوانایی بهتر میشه. اما خب همچین درخواستی از کاربران شاید  باعث بشه که فکر کنن فرایند مشارکت سخت شده و بیخیال مشارکت بشن. البته شاید هم اینطور نباشه و من اشتباه دارم فکر میکنم. ولی خب فکر میکنم این موضوع بهتره بیفته روی دوش ویراستاران خبرنامه و در حد یک سری قواعد و قوانین ساده **درست نوشتن** از کاربر خواسته بشه. با توجه به این که شما در این زمینه به نظر میاد دستی بر آتش دارین، آیا براتون مقدوره یک متن خیلی ساده از نکاتی که باید موقع نوشتن رعایت بشه، آماده کنین؟ در حد ۴ ۵ نکته مهم و کلی. من هم فعلا خودم رو موظف میکنم که هر محتوایی رو که بررسی میکنم، از جهت ویراستاری هم بررسی کنم

</div>

**Reply by ohmydevops** [reply]
خبرنامه در حال حاضر هفتگی ارسال میشه و فعلا من فرصت پیگیری و نگهداری محتوا و کارهای خبرنامه رو دارم. میتونم این موضوع رو پیگیری کنم.

**Reply by ghost** [reply]
<div dir="rtl">
ممنونم.
من ویراستار نیستم ولی به واسطهٔ علاقه‌م به بهترنویسی همیشه سعی کردم اطلاعاتم رو در این حوزه افزایش بدم. به نظرم کتابی نوشتن (فارسی معیار) کار سختی نیست؛ همه باهاش آشنا هستن؛ یعنی مثل نشانه‌های سجاوندی نیست که ذهن رو درگیر کنه. کاربری که تا اینجای راه اومده این یه قلم هم رعایت می‌کنه. به هر حال بازم برای اینکه خیالمون راحت بشه می‌تونیم اینو به عنوان یک قاعدهٔ اختیاری بذاریم که کار شما در مرحلهٔ ویرایش سریع‌تر جلو بره. چون محتوای خبرنامه زیاد نیست خودتون می‌تونید این کار رو انجام بدید. منم اگه فرصتش رو داشتم کمک می‌کنم. به نظرم هر قاعده‌ای به جز فارسی معیار سخت‌گیری به حساب می‌آد و بهتره این کارها رو شخص بازبین انجام بده.
</div>

**Reply by ohmydevops** [reply]
این هم نکات سرجمع و جالبی برای من داشت
https://meidaan.com/how-do-we-write-in-meidaan

**Reply by ghost** [reply]
<div dir="rtl">
به نظرم اگر می‌خواهید قواعد نگارشی بذارید حتماً اختیاری بذارید؛ چون رعایت هر قاعده‌ای زمان می‌بره. شاید طرف دفعهٔ اول یا دوم به خاطر اشتیاقش همکاری کنه ولی دفعات بعد به خاطر وقتی که ازش گرفته شده منصرف می‌شه. برای یک خبرنامهٔ تخصصی، محوریت با اصل محتواست. ویرایش محتوا رو بعداً هم می‌شه انجام داد.
</div>

**Reply by Hameds** [reply]
<div dir='rtl'>

اگر اشتباه املایی در محتوا وجود داره بهتره اصلاح بشه، اما گنجاندن فارسی معیار در قوانین کار درستی نیست. ممکنه خود محتوا به فارسی معیار نوشته نشده باشه و بهتره هر کاربری با ادبیات خودش بتونه محتوا رو معرفی کنه

</div>

**Reply by ohmydevops** [reply]
بله فعلا در حد موضوعات خیلی ساده و ابتدایی رو من خودم موقع تایید محتوا بررسی میکنم و اگر نکته‌ای باشه یادداشت میکنم یا با اجازه خود کاربر ویرایش میکنم. مهم محتواست

**Reply by ghost** [reply]
> اگر اشتباه املایی در محتوا وجود داره بهتره اصلاح بشه، اما گنجاندن فارسی معیار در قوانین کار درستی نیست. ممکنه خود محتوا به فارسی معیار نوشته نشده باشه و بهتره هر کاربری با ادبیات خودش بتونه محتوا رو معرفی کنه

<div dir="rtl">
مگه ما با همین زبان محاوره‌ای به مقالات رسمی یا بالعکس ارجاع نمی‌دیم؟ اینکه چون مقاله‌ای به زبان شکسته نوشته شده پس ما هم باید به زبان شکسته بهش ارجاع بدیم استدلال صحیحی نیست. مقصود من این بود که نباید در یک نوشتار، مدام بین فارسی معیار و محاوره‌ای سوئیچ کنیم که در حال حاضر این اتفاق در خبرنامه می‌افته. من به عنوان یک خواننده تا می‌آم به لحن محاوره‌ای عادت کنم پاراگراف بعدی با نگارش رسمی مواجه می‌شم. مشکلی بزرگی نیست ولی همچنان یک مشکله.
</div>

**Comment by ohmydevops** [comment]
من سعی میکنم یک جمع‌بندی برای قوانین بکنم و کم کم توی سایت هم اضافه کنمش. چون مدت زیادی گذشته و هنوز قوانین رو توی سایت نزاشتیم

**Reply by ghost** [reply]
<div dir="rtl">
شما قوانینی که به ذهنتون می‌رسه رو بنویسید. من تکمیل و ویرایش می‌کنم. ولی پیشنهاد می‌کنم تا چکش‌کاری نشده تو سایت نذارید.
</div>

**Reply by ohmydevops** [reply]
بسیار خوب. بله اول اینجا جمع‌بندی میکنیم

**Comment by ohmydevops** [comment]
<div dir='rtl'>

قوانینی که حامد گردآوری کرده بود به نظرم میتونه نقطه شروع خوبی برای ثبت شدن در سایت باشه. چون مدت زیادی گذشته و هنوز بخش قوانین سایت رو آپدیت نکردیم. من با مفاد قوانین اشتراک محتوایی که حامد گفته موافقم. فعلا این قوانین رو در سایت قرار میدیم و لینک این صفحه رو هم میزاریم تا در صورتی که کسی نظر مخالفی داره یا پیشنهادی داره بتونه مطرحش کنه.

</div>

**Comment by ghost** [comment]
<div dir="rtl">
قوانینِ آقا حامد رو خلاصه‌تر کردم. 

البته موردی رو که گفته «چنانچه محتوای پیشنهادی...» متوجه نشدم.
 
- تاحدامکان عنوان و توضیحِ محتوا کوتاه باشد؛
- تاحدامکان برای محتوایِ غیرفارسی از عنوان و توضیحِ فارسی استفاده کنید؛
- محتوا با هدفِ تبلیغات و بازاریابی پذیرفته نیست؛
- محتوایی که فقط با IP ایران باز می‌شود پذیرفته نیست؛
- محتوایِ تکراری که در خبرنامه‌هایِ پیشین منتشر شده پذیرفته نیست؛
- لینکِ کوتاه پذیرفته نیست؛ مگرآنکه مربوط به سرویس‌دهنده‌یِ خودِ محتوا باشد (مثلاً t.co یا vrgl.ir)؛
- در هر شماره فقط یک محتوا مربوط به صفحه‌یِ شخصیِ شما (ویرگول، بلاگ، یوتیوب و ...) پذیرفته می‌شود؛
- صحت و سقمِ محتوا بر عهده‌یِ نویسنده‌یِ آن است؛
- با درنظرگرفتنِ قوانینِ بالا، به‌هرتعداد می‌توانید محتوا پیشنهاد دهید.
</div>

**Reply by ohmydevops** [reply]
<div dir='rtl'>

ممنون ایوب عزیز. خیلی عالی شد.

</div>

**Comment by ghost** [comment]
<div dir="rtl">
می‌شه قوانین رو در قالبِ یک چک‌لیست در انتهایِ قالبِ خبرنامه قرار داد که مشارکت‌کننده قبل از ثبتِ issue اون‌ها رو تیک بزنه. با اینکار قوانین همیشه جلویِ چشمه و به‌ناچار قبل از ثبتِ پیشنهاد باید چک بشه.

---

تأیید می‌کنم که محتوایِ پیشنهادی:

- در راستایِ موضوعِ خبرنامه است؛
- هدفش تبلیغ و بازاریابی نیست؛
- با IP غیرایرانی باز می‌شود؛
- تکراری نیست (در خبرنامه‌هایِ پیشین منتشر نشده است)؛
- لینکِ کوتاه نیست یا لینکِ کوتاهِ خودِ سرویس‌دهنده‌یِ محتواست (مثلاً t.co‌ برایِ توییتر و vrg.ir برایِ ویرگول)؛
- مربوط به صفحه‌یِ شخصی نیست یا اگر هست فقط محدود به یک محتوا برایِ این شماره از خبرنامه است.
- تا حدِ امکان عنوان و توضیحِ آن فارسی است؛
- تا حدِ امکان عنوان و توضیحِ آن کوتاه است.

صحت و سقمِ محتوایِ پیشنهادی بر عهده‌یِ نویسنده‌یِ آن است. 
</div>

**Reply by ohmydevops** [reply]
این مورد خیلی خوب و کمک‌کننده هستش
من الان دیدمش، عجیبه که نوتیف نیومده بود برام.

**Comment by ohmydevops** [comment]
> این نسخه اولیه پیشنهادی من برای قوانین مشارکت در محتواست.
> 
> * تا حد امکان عنوان و توضیحات محتوای پیشنهادی که در مخزن در قالب `issue` اضافه می‌شود، کوتاه و به زبان فارسی باشند. البته خود محتوا می‌تواند به فارسی یا انگلیسی باشد.
> * لینک محتوای پیشنهادی نباید از سرویس‌های کوتاه‌کننده‌های لینک باشد مگر کوتاه کننده لینک مربوط به سرویس‌دهنده همان محتوا
> * لینک محتوای پیشنهادی می‌بایست از خارج از ایران هم در دسترس باشد، محتوایی که فقط با IP ایران در دسترس است مورد پذیرش نیست
> * در صورتی که سایت محتوای پیشنهادی در حال فروش مستقیم محصول یا خدمات است، محتوا باید خارج از بخش تبلیغاتی و فروش سایت باشد
> * محتوای بازاریابی محصولات نرم‌افزاری مورد تایید برای ارسال در خبرنامه نیست
> * از صفحه هر شخص (مثال ویرگول یا یوتوب یا وبلاگ شخصی و امثالهم) در هر شماره فقط یک محتوا برای ارسال در خبرنامه تایید می‌شود اما محدودیتی در ارسال لینک‌های متعدد نیست
> * لینک یا محتوای تکراری در خبرنامه‌های پیشین مجدداً ارسال نمی‌شود
> * چنانچه محتوای پیشنهادی مربوط به مسائل روابط عمومی یک شرکت است، با صلاحدید مدیران و تنها از وب سایت رسمی آن شرکت امکان پیشنهاد محتوا وجود دارد و نظرات کارمندان در صفحات شخصی معیار پذیرش محتوای مرتبط با روابط عمومی شرکت‌ها نیست
> * بدیهی است مسئولیت صحت اطلاعات در هر محتوای پیشنهادی به عهده نویسنده محتوا می‌باشد.
> * هر کاربری می‌تواند محتواهای ایجاد شده توسط خودش یا دیگران را که شرایط بالا را داشته باشند به تعداد نامحدود در این مخزن معرفی کند

بخشی که گفتیم محتوای تکراری ارسال نمیشود به نظرم نیازی به بیانش نباید باشه. چون وظیفه ما هستش به صورت خودکار بررسی کنیم. کاربر نمیتونه این مورد رو چک کنه پس در عمل قابل انجام نیست. به صورت ماشینی باید بررسی کنیم. لذا از قوانینی که در خود سایت نشون میدیم حذفش کردم.

@Hameds

**Comment by ohmydevops** [comment]
قوانین به سایت اضافه شدند
https://github.com/softwaretalks/newsletter/commit/cd39f7c4488b67693eec2100e4610a29d1c4096b
